src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java

```diff
@@ -126,6 +126,15 @@ Integer linkLetterHolder(
         @Bind("letterHolderId") final String letterHolderId
     );
 
+    @SqlUpdate(
+        "UPDATE claim SET sealed_claim_document_management_self_path = :documentSelfPath"
+            + " WHERE id = :claimId"
+    )
+    Integer linkSealedClaimDocument(
```

Comment:
Shouldn't `Long` be returned?

Reply:
many have Integer, they all should be long. however, we don't check for returned value as how many rows updated

Reply:
Return value represents number of updated rows which maybe we well represented by integer. I do not see much value in changing this now unless you give me some good reason.